selfdrive/car/tests/routes.py

```diff
@@ -292,7 +292,6 @@ class CarTestRoute(NamedTuple):
   # Segments that test specific issues
   # Controls mismatch due to interceptor threshold
   CarTestRoute("cfb32f0fb91b173b|2022-04-06--14-54-45", HONDA.CIVIC, segment=21),
-  CarTestRoute("5a8762b91fc70467|2022-04-14--21-26-20", TOYOTA.RAV4, segment=2),
```

Comment:
tested by fuzzing test now, had a legit relay malfunction in this segment (SDSU started publishing again)